kelvin/settings.py

```diff
@@ -31,9 +31,9 @@
 SECRET_KEY = "***REMOVED***"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = bool(os.environ.get("DEBUG", default=0))
```

Comment:
We can keep these two settings as they were, they will be overwritten on production anyway with a `local_settings.py` file (that somehow needs to find its way into the Docker image :) ).

Reply:
Reverted